### PR TITLE
Remove extract fingerprints

### PR DESCRIPTION
**Pull request by gavin-peng**
Test adjusted workflow, prepared vidarr  deploying files.

### CODE REVIEW

File: vidarrtest-regression.json.in [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [
   {
     "id": "TEST_0001_WG_HG38_FASTQ_INPUT",
-    "description": "runs the fingerprintsCollector workflow on a WG library using hg38, with fastq as input",
+    "description": "runs the hotspotFingerprintsCollector workflow on a WG library using hg38, with fastq as input",
```
Comment (lheisler) [inline vidarrtest-regression.json.in:4]: hotspotFingerprintCollector, not hotspotFingerprintsCollector
distinction is tha this workflow will generate a single fingerprint.  The dev workflow that it was based on was meant to generate multiple fingerprints

File: vidarrtest-regression.json.in [hunk 2]
```diff
@@ -27,162 +27,142 @@
...
+      "hotspotFingerprintsCollector.outputFileNamePrefix": "TEST_0001_WG_HG38_FASTQ",
+      "hotspotFingerprintsCollector.aligner": "bwa",
+      "hotspotFingerprintsCollector.markDups" : "true",
+      "hotspotFingerprintsCollector.refFasta": "/.mounts/labs/gsi/modulator/sw/data/hg38-p12/hg38_random.fa",
+      "hotspotFingerprintsCollector.bwaMem.runBwaMem_modules": "samtools/1.9 bwa/0.7.12 hg38-bwa-index-with-alt/0.7.12",
+      "hotspotFingerprintsCollector.bwaMem.runBwaMem_bwaRef": "/.mounts/labs/gsi/modulator/sw/data/hg38-bwa-index-with-alt-0.7.12/hg38_random.fa",
+      "hotspotFingerprintsCollector.bwaMem.adapterTrimmingLog_timeout": null,
+      "hotspotFingerprintsCollector.bwaMem.adapterTrimmingLog_jobMemory": null,
+      "hotspotFingerprintsCollector.bwaMem.indexBam_timeout": null,
+      "hotspotFingerprintsCollector.bwaMem.indexBam_modules": null,
```
Comment (lheisler) [inline vidarrtest-regression.json.in:40]: see note above regarding the name,  these will all fail as they use the incorrect name (fingerprints with an s is incorrect)

**Review by lheisler — CHANGES_REQUESTED** [review]
commented on using the name uniformly
hotspotFingerprintCollector, not
hotspotsFingerprintCollector

also, some of the metadata in the wdl file can be updated.  you can add your name as a developer, i'm also going to re-add Peter who developed the earlier workflow


have you tested the regression tests?  from what i can see they should all fail.  I think you can test in Jenkins on a commit, ie. without a pull request

File: vidarrtest-regression.json.in (hunk 2 above)
Comment (gavin-peng) [reply vidarrtest-regression.json.in]: done!

File: vidarrtest-regression.json.in (hunk 1 above)
Comment (gavin-peng) [reply vidarrtest-regression.json.in]: done

File: hotspotFingerprintCollector.wdl [hunk 3]
```diff
@@ -120,8 +118,8 @@ workflow hotspotFingerprintCollector {
      }
 
     meta {
-     author: "Lawrence Heisler"
-     email: "lawrence.heisler@oicr.on.ca"
+     author: "Lawrence Heisler and Gavin Peng"
+     email: "lawrence.heisler@oicr.on.ca and Gavin.Peng@pinr.on.ca"
```
Comment (lheisler) [inline hotspotFingerprintCollector.wdl:122]: typo on gavin's email address, should be oicr.on.ca

File: tests/calculate.sh [hunk 4]
```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
-set -o nounset
```
Comment (lheisler) [inline tests/calculate.sh:2]: these set lines were removed, i think they are best practices to keep in the calculate.sh script or possibly need to be there for jenkins?

**Review by lheisler — APPROVED** [review]
a few small changes indicated, but otherwise i think fine to merge.

**Review by pruzanov — APPROVED** [review]

File: tests/calculate.sh (hunk 4 above)
Comment (gavin-peng) [reply tests/calculate.sh:2]: I saw once jenkins report one of the error guarding setting (-o pipefail) invalid, feel strange though, I can add back to test again.

**Comment by gavin-peng** [comment]
> a few small changes indicated, but otherwise i think fine to merge.

This version includes the python file in src folder, while still use the that python code in the wdl. If have time to debug, can replace the python code in wdl with calling of the python file.

**Pull request merged**